### PR TITLE
[dev-qt/qtscxml] New package.

### PR DESCRIPTION
**Pull request by tbrethom**
Loosely based on the existing qt packages.

Please feel free to comment.

### CODE REVIEW

File: dev-qt/qtscxml/qtscxml-5.7.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit qt5-build
+
+DESCRIPTION="State chart XML support library for the Qt5 framework."
```
Comment (Pesa) [inline dev-qt/qtscxml/qtscxml-5.7.0.ebuild]: I don't think we have periods in the other descriptions?

File: dev-qt/qtscxml/qtscxml-5.7.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+if [[ ${QT5_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc64 ~x86"
```
Comment (Pesa) [inline dev-qt/qtscxml/qtscxml-5.7.0.ebuild]: here too, start with just `~amd64` and maybe `~x86`

File: dev-qt/qtscxml/qtscxml-5.7.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+fi
+
+DEPEND="
+	~dev-qt/qtcore-${PV}
+	~dev-qt/qtdeclarative-${PV}
+	~dev-qt/qtnetwork-${PV}
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
```
Comment (Pesa) [inline dev-qt/qtscxml/qtscxml-5.7.0.ebuild]: redundant

File: dev-qt/qtscxml/qtscxml-5.7.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+DESCRIPTION="State chart XML support library for the Qt5 framework."
+
+if [[ ${QT5_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc64 ~x86"
+fi
+
+DEPEND="
+	~dev-qt/qtcore-${PV}
+	~dev-qt/qtdeclarative-${PV}
```
Comment (Pesa) [inline dev-qt/qtscxml/qtscxml-5.7.0.ebuild]: can this dep be made optional without too much effort?

Comment (tbrethom) [inline dev-qt/qtscxml/qtscxml-5.7.0.ebuild]: No, it is used in one of the core data models of Qt SCXML (ECMAScript).

**Comment by tbrethom** [comment]
Fixed

File: dev-qt/qtscxml/qtscxml-5.7.0.ebuild (hunk 4 above)
Comment (Pesa) [inline dev-qt/qtscxml/qtscxml-5.7.0.ebuild]: alright

**Comment by Pesa** [comment]
Please add `IUSE=""` in all ebuilds, even if empty, and reformat the commit msg to be in the form `cat/pn: foo` instead of using square brackets. (see the merged qtquickcontrols2 commit for reference)

File: dev-qt/qtscxml/qtscxml-5.7.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit qt5-build
+
+DESCRIPTION="State chart XML support library for the Qt5 framework"
```
Comment (Pesa) [inline dev-qt/qtscxml/qtscxml-5.7.0.ebuild]: `"State Chart XML (SCXML) support library etc..."`

**Comment by Pesa** [comment]
Merged in 0f71baadd639b8a2c2fbb627b847a9c3dc0cd01f after removing bogus dep on qtnetwork. Thanks!

**Pull request closed without merging**